util/Radiance_Monitor/image_gen/ush/RadMon_IG_glb.sh

```diff
@@ -208,8 +208,8 @@ fi
 #
 nfile_src=`ls -l ${ieee_src}/*${PDATE}*ieee_d* | egrep -c '^-'`
 if [[ $nfile_src -le 0 ]]; then
-   if [[ -e ${ieee_src}/radmon_angle.tar ]]; then
-      nfile_src=`tar -tf ${ieee_src}/radmon_angle.tar | grep ieee_d | wc -l`
+   if [[ -e ${ieee_src}/radmon_angle.tar || -e ${ieee_src}/radmon_angle.tar.${Z} ]]; then
+      nfile_src=`tar -tf ${ieee_src}/radmon_angle.tar* | grep ieee_d | wc -l`
```

Comment:
I'm tempted to say it's not possible that both versions exist.  That's my intent.  But you're probably right to suggest some defensive measures just in case.  I'll flag both as an error condition.